Add notices about slow response times to README and Issue templates (#1806)
dependabot/dependabot-core: .github/ISSUE_TEMPLATE/bug-report.md

> ---

Reviewer: Should we sync the content of these up with the ones in feedback?
https://github.com/dependabot/feedback/edit/master/.github/ISSUE_TEMPLATE/1-bug-report.md
https://github.com/dependabot/feedback/edit/master/.github/ISSUE_TEMPLATE/2-feature-request.md

Author: Ah yes nice one 👍

> name: Bug report

Reviewer: ```suggestion
name: "\U0001F41B Bug report"
```

> about: Create a report to help us improve

Reviewer: ```suggestion
about: Report a bug in Dependabot to help us fix it
```

title: ''
labels: bug
assignees: ''

---

> 

Reviewer: ```suggestion

<!-- Please search existing issues to avoid creating duplicates. -->

```

<!-- The Dependabot team is currently at reduced capacity, because of this our
response times on issues will be slower than we'd like. -->

**Describe the bug**
A clear and concise description of what the bug may be.

**Package ecosystem**: <!-- bundler, docker, npm.. etc -->

**Expected behavior**
A clear and concise description of what you expected to happen.

**PR, Issue or logs**
If applicable, add links to public PR's or Issues that Dependabot opened, and/or

> paste in any related logs.

Reviewer: I like this over what we have in the feedback repo. We currently ask for the repo but might be better to start asking for manifest files etc for easier reproduction instead.

Something like?
```
<!-- The more information you can provide, the easier it will be to reproduce the issue and find a fix -->

**Package manage/ecosystem**
<!-- ruby:bundler, javascript, docker.. etc -->
**Manifest contents prior to update**
<!-- If applicable, link to package.json, Gemfile.. etc -->
**Updated dependency**
<!-- If applicable, the dependency name and to and from versions -->
**What you expected to see, versus what you actually saw**
<!-- A clear and concise description of what you expected to happen -->
**Images of the diff or a link to the PR, issue or logs**
<!-- If applicable, add links to public PR's or Issues that Dependabot opened, and/or
paste in any related logs -->
```

Author: Yeah, I like that 👍
